### PR TITLE
Change hacked installment minimum to 200000 subunits

### PR DESCRIPTION
**Pull request by som-m**
https://www.omise.co/installment-payments#limits

### CODE REVIEW

**Review by zentetsukenz — APPROVED** [review]
👍

**Review by rezigned — APPROVED** [review]

File: lib/omise/OmiseCapabilities.php [hunk 1]
```diff
@@ -123,7 +123,7 @@ public function makeBackendFilterChargeAmount($amount)
         $defMin = $this['limits']['charge_amount']['min'];
         $defMax = $this['limits']['charge_amount']['max'];
         return function ($backend) use ($amount, $defMin, $defMax) {
-            // temporary hack for now to correct min value for instalments to fixed minium (different to normal charge minimum)
+            // temporary hack for now to correct min value for installments to fixed minimum (different to normal charge minimum)
```
Comment (danfowler) [inline lib/omise/OmiseCapabilities.php:126]:
```suggestion
            // permanent hack for now to correct min value for installments to fixed minimum (different to normal charge minimum)
```

Comment (danfowler) [reply lib/omise/OmiseCapabilities.php:126]: 😏

**Review by danfowler — APPROVED** [review]

File: lib/omise/OmiseCapabilities.php (hunk 1 above)
Comment (danfowler) [reply lib/omise/OmiseCapabilities.php:126]: You don't have to commit the above :)

**Pull request merged**